external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -55,7 +56,7 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSQLCo
   import testImplicits._
 
   override val db = new DatabaseOnDocker {
-    override val imageName = "wnameless/oracle-xe-11g:16.04"
+    override val imageName = "deepdiver/docker-oracle-xe-11g:2.0"
```

Comment:
Why do we need update this image?

Reply:
hmm.. who owns/licenses these images...

Reply:
> Why do we need update this image?

![image](https://user-images.githubusercontent.com/698621/54331459-0dd8fe80-4655-11e9-9ddc-8869f480e035.png)

Reply:
I see, but is the new image 'legal' either? is there an official one? and if not, should we be using or testing against it?

Reply:
I also have the same concerns on the legal stuff. If we don't have an official docker image, we had better remove this from docker test.

Reply:
Seems there is no official Oracle docker image,  can we Ignore this Test to pass the docker integration test?

Reply:
I think it should be deleted if there is no legitimate way to test this

Reply:
agreed there..

Reply:
Looks like this is how you build it:
https://github.com/oracle/docker-images/blob/master/OracleDatabase/SingleInstance/README.md
but looks like indeed you have to supply the binaries as they're not freely available.
Yes at best I think we'd need to disable this test and parameterize it, as it's not going to work reliably anyway.